webrtc.html

```diff
@@ -1243,6 +1243,14 @@
                         "RTCRtpTransceiver">mid</a></code> value to the mid of
                         the corresponding media description.</p>
                       </li>
+                      <li>
+                        <p>If <var>description</var> is of type "answer", then
+                        set <var>transceiver</var>'s [[<a href=
+                        "#curr-direction-slot">CurrentDirection</a>]] slot
```

Comment:
Thanks for making an issue. This has been brought up at the editor's meeting as well.